sdks/python/apache_beam/runners/portability/fn_api_runner.py

```diff
@@ -1365,6 +1365,10 @@ def start_worker(self):
   def stop_worker(self):
     pass
 
+  def localhost_from_worker(self):
```

Comment:
Could we rename this method here and elsewhere?

```suggestion
  def host_from_worker(self):
```